bin/deploy.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
One thing I saw Michael W and An doing was using `#!/usr/bin/env bash` instead of `#!/bin/bash` for shell scripts and thought I would pass on the option. I think the main motivation is it's more portable since it searches the PATH for bash

Reply:
Thanks @alexzbusko I was not aware of that and I will make sure I use that in any bash scripts moving forward.